cmd/utils/flags.go

```diff
@@ -2081,6 +2086,8 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 			cfg.EthDiscoveryURLs = SplitAndTrim(urls)
 		}
 	}
+	CheckExclusive(ctx, ECBP1100DisableFlag, ECBP1100NoDisableFlag)
```

Comment:
@meowsbits this is probably better to be removed (I added it).

A node operator might indeed want the `nodisable` functionality, so if he still wants to enable ECBP-1100 he has to do `--ecbp1100.disable=69420123 --ecbp1100.nodisable`.

The thing is that with the above check, this is not allowed.